contents/docs/surveys/implementing-custom-surveys.mdx

```diff
@@ -83,4 +83,50 @@ posthog.capture("survey sent", {
     $survey_id: {survey.id} // required
     $survey_response: {survey_response} // required
```

Comment:
I think it would shorter and simpler to add a comment here explaining what the value for survey_response should be. What do you think?

```suggestion
    $survey_response: {survey_response} // required. `{survey_response}` must be a text or number value (depending on your question type).
    // For multiple choice select surveys, `{survey_response}` must be an array of values with the selected choices.
    // e.g., $survey_response: ["response_1", 8, "response_2"]
```

Reply:
Sure! I just want to make sure that users are aware of their response values so they can make the most of the automatic survey results graphs etc